app/menus/DocumentMenu.tsx

```diff
@@ -217,160 +212,164 @@ function DocumentMenu({
         </label>
       </VisuallyHidden>
       {label ? (
-        <MenuButton {...menu}>{label}</MenuButton>
+        <MenuButton {...menu} onMouseEnter={handleMouseEnter}>
+          {label}
+        </MenuButton>
       ) : (
         <OverflowMenuButton
           className={className}
           aria-label={t("Show menu")}
+          onMouseEnter={handleMouseEnter}
           {...menu}
         />
       )}
-      <ContextMenu
-        {...menu}
-        aria-label={t("Document options")}
-        onOpen={onOpen}
-        onClose={onClose}
-        loading={loading || documents.isFetching}
-      >
-        <Template
+      {data && !loading && !documents.isFetching ? (
```

Comment:
`documents.isFetching` this would prevent the menu from showing when _any_ fetch is happening…

Reply:
Ahh! Good catch, thanks 🙏